cob_bringup/robots/cob4-7.xml

```diff
@@ -128,11 +128,9 @@
 			<arg name="sim" value="$(arg sim)"/>
 			<arg if="$(arg sim)" name="cartesian_control" value="true"/>
 		</include>
-		<include file="$(find cob_bringup)/components/cam3d_openni2.launch">
+		<include file="$(find cob_bringup)/components/cam3d_zr300_rgbd.launch">
 			<arg name="robot" value="$(arg robot)"/>
-			<arg name="name" value="torso_cam3d_right"/>
-			<arg name="device_id" value="#1"/>
-			<arg name="data_skip" value="0"/>
+			<arg name="name" value="torso_cam3d_left"/>
```

Comment:
why has the sides been swapped?
cam_left is now on t1 - cam_right on t3
the other robots have it the other way! can we re-plug the cameras?